README.md

```diff
@@ -1,295 +1,397 @@
+
+The same logic could be implemented in NodeJS using the same JS API:
+
+```js
+global.request = require("then-request");
+var JanusVideoRoom = require("./html/janusvideoroom.js");
+var janus = new JanusVideoRoom(
+  "http://192.168.0.15:8088/janus",
+  "http://192.168.0.15:8000",
+);
+janus.join(1234, "mmal service 16.1", "video");
+```
+
+## Connect to Jitsi
+
+A simple way to publish WebRTC stream to a [Jitsi](https://meet.jit.si) Video
+Room is to use the
+[XMPPVideoRoom](http://htmlpreview.github.io/?https://github.com/mpromonet/webrtc-streamer-html/blob/master/jsdoc/XMPPVideoRoom.html)
+interface
+
+```js
+var xmpp = new XMPPVideoRoom(<xmpp server url>, <webrtc-streamer url>)
+```
+
+A short sample to publish WebRTC streams to a Jitsi Video Room could be:
+
+```html
+<html>
+<head>
+<script src="libs/strophe.min.js" ></script>
+<script src="libs/strophe.muc.min.js" ></script>
+<script src="libs/strophe.disco.min.js" ></script>
+<script src="libs/strophe.jingle.sdp.js"></script>
+<script src="libs/jquery-3.5.1.min.js"></script>
+<script src="xmppvideoroom.js" ></script>
+<script>        
+	var xmpp = new XMPPVideoRoom("meet.jit.si", null);
+	xmpp.join("testroom", "rtsp://wowzaec2demo.streamlock.net/vod/mp4:BigBuckBunny_115k.mov","Bunny");	    
+</script>
+</head>
+</html>
+```
 
 [Live Demo](https://webrtcstreamer.agreeabletree-365b9a90.canadacentral.azurecontainerapps.io/xmppvideoroom.html)
 
-Docker image
-===============
-You can start the application using the docker image :
-
-        docker run -p 8000:8000 -it mpromonet/webrtc-streamer
-
-You can expose V4L2 devices from your host using :
-
-        docker run --device=/dev/video0 -p 8000:8000 -it mpromonet/webrtc-streamer
+## Docker Image
 
-The container entry point is the webrtc-streamer application, then you can :
+You can start the application using the docker image:
 
-* get the help using :
+```sh
+docker run -p 8000:8000 -it mpromonet/webrtc-streamer
+```
 
-        docker run -p 8000:8000 -it mpromonet/webrtc-streamer -h
+You can expose V4L2 devices from your host using:
 
-* run the container registering a RTSP url using :
+```sh
+docker run --device=/dev/video0 -p 8000:8000 -it mpromonet/webrtc-streamer
+```
 
-        docker run -p 8000:8000 -it mpromonet/webrtc-streamer -n raspicam -u rtsp://pi2.local:8554/unicast
+The container entry point is the webrtc-streamer application, then you can:
 
-* run the container giving config.json file using :
+- view all commands
+  ```sh
+  docker run -p 8000:8000 -it mpromonet/webrtc-streamer --help
+  ```
+- run the container registering a RTSP url:
 
-        docker run -p 8000:8000 -v $PWD/config.json:/app/config.json mpromonet/webrtc-streamer
+  ```sh
+  docker run -p 8000:8000 -it mpromonet/webrtc-streamer -n raspicam -u rtsp://pi2.local:8554/unicast
+  ```
+- run the container giving config.json file:
 
+  ```sh
+  docker run -p 8000:8000 -v $PWD/config.json:/app/config.json mpromonet/webrtc-streamer
+  ```
 
+## License
```

Comment:
LICENSE is already presented by github in the side panel.

Reply:
Oops -- yeah, that's redundant, removing now.